static/app/components/assistant/types.tsx

```diff
@@ -38,12 +56,21 @@ type BaseGuide = {
 };
 
 export type Guide = BaseGuide & {
+  /**
+   * Has this guide already been seen?
+   */
   seen: boolean;
 };
 
 export type GuidesContent = BaseGuide[];
 
 export type GuidesServerData = {
+  /**
+   * Guide key
+   */
   guide: string;
+  /**
+   * Has the guide been seen
```

Comment:
shouldn't we use the same comment here?

```
/**
   * Has this guide already been seen?
   */
```

Reply:
yeah will fix these two small ones, have another PR that depends on these changes tho :)

Reply:
Oh, no approval, i'll make the changes then :)